rpm/stackdriver-agent.spec

```diff
@@ -110,6 +111,8 @@ BuildRequires: hiredis-devel
 %endif
 %if %{has_yajl}
 BuildRequires: yajl-devel
+%endif
+%if %{has_yajl} && ! %{bundle_yajl}
```

Comment:
You can fold this into the previous `%if %{has_yajl}` and avoid the `&&`.
That said, you need to initialize `%{bundle_yajl}` via `%define bundle_yajl 0` around line 17.

Reply:
It worked. Confirmed on Centos 6 and 7.

Reply:
My main concern was actually Amazon Linux... :smile: